valhalla/tyr/route_serializer_osrm.h

```diff
@@ -0,0 +1,44 @@
+#ifndef VALHALLA_TYR_ROUTE_SERIALIZER_OSRM_
+#define VALHALLA_TYR_ROUTE_SERIALIZER_OSRM_
+#pragma once
+
+#include "proto_conversions.h"
+
+namespace osrm_serializers {
```

Comment:
I can't say much to this, not sure why it was a private header/source file before. IMO it's good to separate it, but maybe there was some logic behind it initially @kevinkreiser ?

Reply:
sometimes it's fine to not have a header I think, as e.g. `src/tyr/matrix_serializer.cc` which just implements smth from `valhalla/tyr/serializers.h`. what I really don't like is to **include** implementation files. but I might not see some hidden benefit of this practice?